starter-code/src/viking.js

```diff
@@ -1,11 +1,121 @@
 // Soldier
-class Soldier {}
+class Soldier {
+    constructor(health, strength){
+        this.health = health;
+        this.strength = strength;
+    }
+
+    attack(){
+        return this.strength;
+    }
+
+    receiveDamage(damage){
+        this.health = this.health - damage;
+    }
+
+}
+
 
 // Viking
-class Viking {}
+class Viking extends Soldier{
+    constructor(name, health, strength){
+        super(health, strength);
+        this.name = name;
+        this.health = health;
```

Comment:
When you use super, you don't have to include this.x = x; the super automatically does that for you